### PR TITLE
major update of WPF community feedback topic

### LINKED ISSUE #4133: FW link on WPF Community Feedback page leads to non-existent site
#fw link on WPF Community Feedback page leads to non-existent site

This document: https://github.com/dotnet/docs/blob/master/docs/framework/wpf/getting-started/community-feedback.md

Issue:  Product Feedback Center link (http://go.microsoft.com/fwlink/?LinkId=99381) leads to a non-existent Connect site.   Determine if we have a better place for WPF product feedback and update the FW link with that if possible.

John Eddy at MS reported this issue since they received complaints to @MSDev.

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #4133 

@SamBent please review

### CODE REVIEW

File: docs/framework/wpf/getting-started/community-feedback.md [hunk 1]
```diff
@@ -1,192 +1,124 @@
...
+
+#### Use meaningful titles
+
+Concise, meaningful titles improve the discoverability of your posts. They also make it easier for other WPF forum community members to determine if they can solve your problem.
+
+#### Include appropriate content
+
+Describe the issue and how you’ve tried to work through it. If possible, include supporting code snippets, or the simplest possible sample that demonstrates your issue. All these details help to increase the chance your question will be answered quickly.
+
+## Visual Studio Developer Community
```
Comment (BillWagner) [inline docs/framework/wpf/getting-started/community-feedback.md]: nit: Should this be sentence case?

**Review by BillWagner — APPROVED** [review]
I found one nit, but otherwise, this LGTM.

Once you've got all the feedback you need, go ahead and :shipit:

**Review by rpetrusha — APPROVED** [review]
LGTM. Once Bill's comment is addressed, this is good to merge.

File: docs/framework/wpf/getting-started/community-feedback.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/wpf/getting-started/community-feedback.md]: That's the actual name of the site so I think it's ok like this. But I found another heading that should be sentence case and it's not.

**Pull request merged**